pkg/reconciler/mongodb/mongodbsource_test.go

```diff
@@ -39,8 +46,39 @@ import (
 	. "knative.dev/pkg/reconciler/testing"
 )
 
+var (
+	sinkDNS    = "sink.mynamespace.svc." + utils.GetClusterDomainName()
+	sinkURI, _ = apis.ParseURL("http://" + sinkDNS)
+
+	sinkDestURI = duckv1.Destination{
+		URI: apis.HTTP(sinkDNS),
+	}
+	sinkDest = duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Name:       sinkName,
+			Namespace:  testNS,
+			Kind:       "Channel",
+			APIVersion: "messaging.knative.dev/v1beta1",
+		},
+	}
+)
+
 const (
-	testRAImage = "github.com/googleinterns/knative-source-mongodb/test/image"
+	image          = "image"
+	RAImage        = "github.com/googleinterns/knative-source-mongodb/test/image"
```

Comment:
nit: don't start with variable names in capital. Maybe just name it testRAImage as before